### DISCUSSION TITLE
Disk Usage - Home Assistant OS VM

### DISCUSSION THREAD

**Discussion by Lockie85** [tteck/Proxmox#2584] [Q&A] [answered]
Hi,

I setup my Home Assistant based on the "Home Assistant OS VM" helper script so I'm hoping I could get some guidance here. After some time I appear to be out of disk space. While I do use some disk heavy applications like Frigate NVR, I keep having to extend the disk size and I'm now up to 400GB! I've tried actions like those listed here (https://www.home-assistant.io/more-info/free-space) but there's not been much of an impact.

Is there a easy method of viewing where disk space is being used, perhaps similar to the visual methods on Windows/Linux (WinDirStat,TreeSize  / Baobab, KDirstat). Or perhaps a good solution to identifying where all my space is going?

Many Thanks

**Comment by Bashewa** [comment]
You can use ncdu

Open terminal in HA add it with 
apk add ncdu

Usage ncdu /somefolder or ncdu / 
Toggle i for info and q to quit

**Comment by DomdomGithub** [comment, marked as answer]
I had the same problem with HA OS. My 500GB disk was nearly full.
In the terminal, type `du -hd 1 /`
It will return the folders sizes. 
In my case, the backup directory was the one causing the problem with old files not deleted.
`cd /root/backup`  to go to the backup folder
`ls -lhS`	will list the files by size
`rm -rf /root/backup/*`	will delete all this crap (source: [https://community.home-assistant.io/t/disk-space-being-eaten-up/481013/7] )
Have fun
